Improve error message

chipironcin opened the pull request:
**\- What I did**
Add single quotes around container name and container id to improve copy-paste avoiding the extra `.` that slips into the clipboard

**\- How I did it**
Adding the single quotes directly editing the file

**\- How to verify it**
docker run --name busybox_container busybox echo hello
docker run --name busybox_container busybox echo hello
Check error message

**\- Description for the changelog**
Add single quotes around container name and container id to improve copy-paste avoiding the extra `.` that slips into the clipboard

**\- A picture of a cute animal (not mandatory but encouraged)**

Add single quotes around container name and container id to improve copy-paste avoiding the extra `.` that slips into the clipboard

Signed-off-by: Jorge Marin chipironcin@users.noreply.github.com

chipironcin commented on daemon/names.go:
```diff
@@ -72,7 +72,7 @@ func (daemon *Daemon) reserveName(id, name string) (string, error) {
 				return "", err
 			}
-			return "", fmt.Errorf("Conflict. The container name %q is already in use by container %s. You have to remove (or rename) that container to be able to reuse that name.", name, id)
+			return "", fmt.Errorf("Conflict. The container name '%q' is already in use by container '%s'. You have to remove (or rename) that container to be able to reuse that name.", name, id)
```
mmmm I was afraid of that, I tried to look for the place where they are added but cannot find it. I guess we'd better use single quotes in both places or none

tonistiigi replied:
I'm not sure if this is the answer you are looking for but its added by stdib fmt https://github.com/golang/go/blob/964639cc338db650ccadeafb7424bc8ebb2c0f6c/src/fmt/format.go#L407-L422

thaJeztah replied:
or try https://play.golang.org/p/EMI-6sMC16

cpuguy83 replied:
`%q` does some escaping too, though, so it can make the output weird.

chipironcin commented:
>%q already adds quotes
 
>mmmm I was afraid of that, I tried to look for the place where they are added but cannot find it. I guess we'd better use single quotes in both places or none

anyone?

thaJeztah commented:
@chipironcin either way is fine with me, what's  the reason you don't like the double quotes added by `%q` though?

mlaventure commented:
ping @chipironcin

chipironcin commented:
Hey guys! I am sorry I have had no time to check this back.
I understand now what is happening (didnt realize `%q = %quoted`)
I prefer single quotes rather than double quotes but keeping in mind `%q` already does the quoting for us maybe we'd better change `%s` for `%q` as @MHBauer 

But @cpuguy83 raised a concern about escaped characters, anyone knows anything about the escape mapping for`%q`?

cpuguy83 commented:
We can just change both to `%q`. Strange that it's alternated.

GordonTheTurtle commented:
<!-- AUTOMATED:POULE:DCO-EXPLANATION -->
Please sign your commits following these rules:
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "master" git@github.com:chipironcin/docker.git somewhere
$ cd somewhere
$ git rebase -i HEAD~842353721808
editor opens
change each 'pick' to 'edit'
save the file and quit
$ git commit --amend -s --no-edit
$ git rebase --continue # and repeat the amend for each commit
$ git push -f
~~~

Amending updates the existing PR. You **DO NOT** need to open a new one.

thaJeztah reviewed (changes requested):
thanks! changes look good to me, but can you squash your commits? You can check our [contributors guide](https://docs.docker.com/opensource/workflow/work-issue/#/pull-and-rebase-frequently) if you need instructions 👍

chipironcin committed:
Use quoted form of container name and container id

Use quoted form of container name and container id to improve copy-paste avoiding the extra `.` that slips into the clipboard

Signed-off-by: Jorge Marin <chipironcin@users.noreply.github.com>

AkihiroSuda reviewed (approved):
LGTM if green

GordonTheTurtle commented:
<!-- AUTOMATED:POULE:DCO-EXPLANATION -->
Please sign your commits following these rules:
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "master" git@github.com:chipironcin/docker.git somewhere
$ cd somewhere
$ git rebase -i HEAD~842354206368
editor opens
change each 'pick' to 'edit'
save the file and quit
$ git commit --amend -s --no-edit
$ git rebase --continue # and repeat the amend for each commit
$ git push -f
~~~

Amending updates the existing PR. You **DO NOT** need to open a new one.

cpuguy83 commented:
Needs a squash

cpuguy83 commented:
Also would be really helpful to re-word your commit message so that the commit title fits within the normal title length, then you can add more details in the commit message body.

GordonTheTurtle commented:
<!-- AUTOMATED:POULE:DCO-EXPLANATION -->
Please sign your commits following these rules:
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "master" git@github.com:chipironcin/docker.git somewhere
$ cd somewhere
$ git rebase -i HEAD~842354317888
editor opens
change each 'pick' to 'edit'
save the file and quit
$ git commit --amend -s --no-edit
$ git rebase --continue # and repeat the amend for each commit
$ git push -f
~~~

Amending updates the existing PR. You **DO NOT** need to open a new one.

thaJeztah commented:
@chipironcin looks like your sign-off got lost during the rebase, can you fix that? Let us know if you need more information on how to do that

chipironcin commented:
Yea, I am sorry about the number of updates this thread is getting. Trying to tie all things together and meet requirements is not being that easy though (i.e. squash two commits separated by a lot of other commits while rebasing from a diferent remote 😛 )

thaJeztah reviewed (approved):
LGTM

mlaventure commented:
Looks like we hit the flakyness of `TestBuildContextDirIsSymlink` restart janky one more time for good mesure

mlaventure reviewed (approved):
LGTM

chipironcin commented:
Finally, thank for your awesome support and review guys! Keep up the good work!

thaJeztah commented:
Yes, it took a bit longer than expected, thanks for contributing!
